### ISSUE TITLE
Bug: Multiple FieldsetTabOpen per Template -> Uncaught Error

### ISSUE THREAD

**Issue by hettiger** [ryancramerdesign/ProcessWire#1976]
It is happening on page save.

## Error Message
```
Error: Uncaught Error: Call to a member function add() on null in /path/wire/modules/Process/ProcessPageEdit/ProcessPageEdit.module:629
Stack trace:
#0 [internal function]: ProcessWire\ProcessPageEdit->___buildForm(Object(ProcessWire\InputfieldForm))
#1 /path/wire/core/Wire.php(347): call_user_func_array(Array, Array)
#2 /path/wire/core/WireHooks.php(548): ProcessWire\Wire->_callMethod('___buildForm', Array)
#3 /path/wire/core/Wire.php(370): ProcessWire\WireHooks->runHooks(Object(ProcessWire\ProcessPageEdit), 'buildForm', Array)
#4 /path/wire/modules/Process/ProcessPageEdit/ProcessPageEdit.module(197): ProcessWire\Wire->__call('buildForm', Array)
#5 [internal function]: ProcessWire\ProcessPageEdit->___execute()
(line 629 of /path/wire/modules/Process/ProcessPageEdit/ProcessPageEdit.module) 

This error message was shown because: site is in debug mode. ($config->debug = true; => /site/config.php). Error has been logged. Administrator has been notified.
```

**Comment by ryancramerdesign** [comment]
What version of PW? Assuming it's the same issue we saw last week, it
should be fixed in 3.0.30. Please let me know if you are still seeing it in
that version. Thanks.

**Comment by hettiger** [comment]
Oh snap you're right. I'm sorry it was version 3.0.29 devns.
